backend/cinema_service/settings.py

```diff
@@ -88,10 +90,10 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "HOST": os.environ["POSTGRES_HOST"],
-        "NAME": os.environ["POSTGRES_DB"],
-        "USER": os.environ["POSTGRES_USER"],
-        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
+        "HOST": os.environ.get("localhost"),
+        "NAME": os.environ.get("default_db_name"),
+        "USER": os.environ.get("default_user"),
+        "PASSWORD": os.environ.get("default_password"),
```

Comment:
why u changed it?
let it be named liked in env_sample
![image](https://user-images.githubusercontent.com/86322722/228320610-a755f184-b31d-4144-9318-f40a29ff6c10.png)